scripts/gen_api_docs.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+#
+# For each file found by `find`, generate a entry on the toctree in
+# `docs/api.rst`.
+#
+# NOTE: This script should be run from the repository docs directory. That is,
+# this script should be run from this script's ../docs directory.
+
+set -euo pipefail
+
+# Recreate the api toctree document with its header
+cat >"api.rst" <<EOF
+API Documentation
```

Comment:
So this will overwrite anything that is in the file at the moment?

Reply:
Yes, the idea is to have it generated entirely by the script